app/core/room.py

```diff
@@ -6,6 +6,8 @@ def __init__(self,roomId,lock):
         self.roomId = roomId
         self.lock = lock
 
+<<<<<<< HEAD
```

Comment:
@MonsieurPetion you have merge conflict artifacts here, there is no way this application or the tests are running on this branch.